balboa-http/src/main/scala/com/socrata/balboa/server/rest/EntitiesRest.scala

```diff
@@ -18,14 +18,18 @@ object EntitiesRest {
 
   def apply(params: Params): ResponseWithType = {
     val predicate: (String => Boolean) =
-      params.get("filter").map(Pattern.compile).map { pattern => (str: String) => pattern.matcher(str).matches }.getOrElse(_ => true)
+      params.get("filter").map(Pattern.compile)
+        .map { pattern => (str: String) => pattern.matcher(str).matches }.getOrElse(_ => true)
 
     val it = ds.entities().asScala.filter(predicate)
     // backwards-compatibility: -1 == no limit
     val limitString = params.getOrElse("limit", "-1")
     val limit = Try(limitString.toInt).recover({
       case err: NumberFormatException =>
-        return ResponseWithType(json, BadRequest(Error(400, "Unable to parse limit : " + JString(err.toString).toString.drop(1).dropRight(1))))
+        return ResponseWithType(json,
+          BadRequest(Error(
+            BadRequestCode,
+            "Unable to parse limit : " + JString(err.toString).toString.drop(1).dropRight(1))))
```

Comment:
If you do get rid of JString, could you check if rojoma json is a necessary dependency of balboa-http?